package.json

```diff
@@ -66,46 +66,45 @@
     "expand-braces": "^0.1.2",
     "expose-loader": "^0.7.5",
     "express": "^4.16.3",
-    "express-extension-to-accept": "0.0.2",
     "express-minify": "^1.0.0",
     "file-loader": "^1.1.11",
     "flot": "^0.8.0-alpha",
-    "forever": "^0.15.3",
     "jquery": "^3.3.1",
     "jquery-ui-bundle": "^1.12.1-migrate",
     "jquery.tooltips": "^1.0.0",
     "js-storage": "^1.0.4",
     "jsonwebtoken": "^8.3.0",
     "lodash": "^4.17.10",
     "long": "^3.2.0",
+    "mime": "^2.3.1",
     "minimed-connect-to-nightscout": "^1.1.1",
     "moment": "^2.22.2",
     "moment-timezone": "^0.5.21",
     "mongodb": "~3.0.11",
     "mongomock": "^0.1.2",
-    "mqtt": "^0.3.13",
+    "mqtt": "^2.18.3",
     "node-cache": "^4.2.0",
     "parse-duration": "^0.1.1",
     "prettyjson": "^1.2.1",
     "pushover-notifications": "^0.2.4",
     "random-token": "0.0.8",
     "request": "^2.87.0",
     "sgvdata": "git://github.com/ktind/sgvdata.git#wip/protobuf",
-    "share2nightscout-bridge": "git://github.com/bewest/share2nightscout-bridge.git#wip/generalize",
+    "share2nightscout-bridge": "git://github.com/PieterGit/share2nightscout-bridge.git#wip/generalize",
```

Comment:
Can you PR the fixes to a new branch in https://github.com/nightscout/share2nightscout-bridge instead?

Reply:
done, https://github.com/nightscout/share2nightscout-bridge/pull/28